drone-webdriver/src/main/java/org/jboss/arquillian/drone/webdriver/configuration/WebDriverConfiguration.java

```diff
@@ -119,12 +119,11 @@ public void setBrowser(String browser) {
     }
 
     public Capabilities getCapabilities() {
-        // return a merge of original capability plus capabilities user has specified in configuration
-        // safely ignore null value here
-        return new DesiredCapabilities(new DesiredCapabilities(
-            _browser.getRawCapabilities() == null ? new HashMap<String, Object>()
-                : _browser.getRawCapabilities()),
-            new DesiredCapabilities(this.capabilityMap));
+        MutableCapabilities mutableCapabilities = new MutableCapabilities(this.capabilityMap);
+        if (_browser.getRawCapabilities() != null) {
+            mutableCapabilities.merge(new ImmutableCapabilities(_browser.getRawCapabilities()));
```

Comment:
AFAIK a new instance is created.
```suggestion
            mutableCapabilities = mutableCapabilities.merge(new ImmutableCapabilities(_browser.getRawCapabilities()));
```